packages/@webex/plugin-meetings/src/multistream/remoteMediaGroup.ts

```diff
@@ -66,6 +67,38 @@ export class RemoteMediaGroup {
     return [...this.unpinnedRemoteMedia, ...this.pinnedRemoteMedia];
   }
 
+  /**
+   * fixCsis
+   *
+   */
+  public fixCsis(
+    remoteMediaCsis: {remoteMedia: RemoteMedia; csi?: number | undefined}[],
```

Comment:
you don't need the `| undefined` bit in the csi type because you already have the `?`